### PR TITLE
Hardware rev b

### PR DESCRIPTION
**Pull request by droidicus**

### CODE REVIEW

**Comment by mlyle** [comment]
Awesome.  I like kicad's diff-- it's easy to tell the approximate amount changed and the areas changed.  I have not viewed this in detail yet but will endeavor to review soon.

Still no push-push love?

Also I was thinking-- we can remove or de-emphasize that nRST pad.  I did not need it to program the boards initially.  It is necessary to replace the bootloader, but so far the lager loader seems reliable to me.

**Comment by droidicus** [comment]
The push-push is different enough it would require nearly a total re-layout. I also chatted with a few others, and we are all concerned about failure/ejection in a crash. I think the flipping onto the pins problem can be solved easily enough (even though people really shouldn't be changing the card while the power is on anyway in this system), with a square of 4 mil Kapton on the top piece.

I will change the nRST pad to a TP and update the PR.

File: hardware/openLager.sch [hunk 1]
```diff
@@ -928,12 +928,12 @@ L LED D1
...
-F 1 "APHHS1005CGCK" V 3900 7850 50  0000 C CNN
+F 1 "LG L29K-F2J1-24" V 3900 7900 50  0000 C CNN
 F 2 "openLagerFootprints:LED_0603" H 4050 7500 50  0001 C CNN
-F 3 "http://datasheet.octopart.com/APHHS1005CGCK-Kingbright-datasheet-17019150.pdf" H 4050 7500 50  0001 C CNN
-F 4 "Kingbright" H 4050 7500 60  0001 C CNN "Mfg Name"
-F 5 "APHHS1005CGCK" H 4050 7500 60  0001 C CNN "Mfg PN"
-F 6 "https://octopart.com/aphhs1005cgck-kingbright-5355571" H 4050 7500 60  0001 C CNN "Link"
+F 3 "http://datasheet.octopart.com/LG-L29K-F2J1-24-Osram-datasheet-41480516.pdf" H 4050 7500 50  0001 C CNN
+F 4 "Osram Opto" H 4050 7500 60  0001 C CNN "Mfg Name"
+F 5 "LG L29K-F2J1-24" H 4050 7500 60  0001 C CNN "Mfg PN"
```
Comment (mlyle) [inline hardware/openLager.sch:935]: Nice, diffused :) Maybe my eyes won't bleed

**Comment by mlyle** [comment]
Looks great :)

**Pull request merged**